core/src/main/java/com/linecorp/armeria/server/HttpServerHandler.java

```diff
@@ -539,7 +540,8 @@ private ChannelFuture respond(ServiceRequestContext reqCtx, boolean addKeepAlive
 
         final ResponseHeaders immutableResHeaders = resHeaders.build();
         ChannelFuture future = responseEncoder.writeHeaders(
-                req.id(), req.streamId(), immutableResHeaders, !hasContent);
+                req.id(), req.streamId(), immutableResHeaders, !hasContent,
+                HttpHeaders.of(), HttpHeaders.of(), false, false, null);
```

Comment:
nit: `false, false, null` are hard to understand. 😅 Add comment to null and false value for readability?
```suggestion
                HttpHeaders.of(), HttpHeaders.of(), false /* enableServerHeader */, false /* enableDateHeader */, null /* sessionProtocol */);
```

Reply:
Thanks, I fixed it.